apollo-router/src/plugins/connectors/plugin.rs

```diff
@@ -44,36 +47,56 @@ impl Plugin for Connectors {
             );
         }
 
-        Ok(Connectors { debug_extensions })
+        let max_requests = init
+            .config
+            .max_requests_per_operation_per_source
+            .or(std::env::var(CONNECTORS_MAX_REQUESTS_ENV)
```

Comment:
Do we document anywhere that the env var is second to the config? I'm not sure if it would be obvious to users that it can't be overwritten with an env var.

Reply:
The documentation isn't written yet, but I've added this to the issue for writing the documentation.